test/unit/BabelPluginExportToFunctionSpec.js

```diff
@@ -72,7 +95,17 @@ test('BabelPluginExportToFunction should inject config ast', (t) => {
 test('BabelPluginExportToFunction should throw on invalid export', (t) => {
     const exportToFunction = new BabelPluginExportToFunction(babelTypes.stringLiteral('testconfig'));
     const code = 'function myRule() { exports.default = function (test) {}; }';
-    t.throws(() => babel.transform(code, { plugins: [ exportToFunction ] }));
+    const error = t.throws(() => babel.transform(code, { plugins: [ exportToFunction ] }));
+
+    t.is(error.message, 'unknown: auth0-bundler is missing default export in module');
+});
+
+test('BabelPluginExportToFunction should throw on variable export', (t) => {
+    const exportToFunction = new BabelPluginExportToFunction(babelTypes.stringLiteral('testconfig'));
+    const code = 'const a = "test"; exports.default = a;';
+    const error = t.throws(() => babel.transform(code, { plugins: [ exportToFunction ] }));
+
+    t.is(error.message, 'unknown: auth0-bundler is missing default export in module');
```

Comment:
I’m not sure if I understand what the error message tries to explain.
Isn’t the error that `auth0-bundler` is not able to resolve a function that is exported?
If so I would suggest something like this:

`Error: auth0-bundler could not resolve a function that is being exported.`